db/data/20210521194339_create_medication_reminder_experiment.rb

```diff
@@ -0,0 +1,13 @@
+class CreateMedicationReminderExperiment < ActiveRecord::Migration[5.2]
+  def up
+    if CountryConfig.current_country?("India")
+      experiment = Experimentation::Experiment.create!(name: "covid medication reminders", experiment_type: "medication_reminder", state: "new")
+      treatment_group = experiment.treatment_groups.create!(description: "one-off message")
+      treatment_group.reminder_templates.create!(message: "notifications.covid.medication_reminder", remind_on_in_days: 0)
```

Comment:
Does `notifications.covid.medication_reminder` exist where it should? Where can I check this?

Reply:
I see them in https://github.com/simpledotorg/simple-server/pull/2542/files. We're good 👍